dash/development/_r_components_generation.py

```diff
@@ -368,7 +371,11 @@ def write_class_file(name,
         project_shortname,
         prefix
     )
-    file_name = "{}{}.R".format(prefix, name)
+
+    if prefix:
+        file_name = "{}{}.R".format(prefix, name)
+    else:
+        file_name = "{}.R".format(name[0].lower() + name[1:])
```

Comment:
Is it intentional that this is not camelCased but the function name is? Also would be nice to 🌴 use the same code for `.R` and `.Rd`

Reply:
fixed in f3b1888c7783be39d9ff4e307a92437177208a4b

Reply:
To summarize a conversation @rpkyle and I had on slack, there's still something to do here:
- filenames will match function names exactly, plus the extension.
- we will convert snake_case to camelCase no matter what (even if, in principle, component names should already be CamelCase because they're js class names).
- we will NOT lowercase `prefix` - if someone wants an uppercase or mixed case prefix, they can have it.

Reply:
> * filenames will match function names exactly, plus the extension.

fixed in a031129

> * we will convert snake_case to camelCase no matter what (even if, in principle, component names should already be CamelCase because they're js class names).

fixed in f639cd2

> * we will NOT lowercase prefix - if someone wants an uppercase or mixed case prefix, they can have it.

fixed in 7ff6730